src/components/Widgets/OSMDataWidget/OSMDataWidget.jsx

```diff
@@ -0,0 +1,33 @@
+import { Component } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { WidgetDataTarget, registerWidgetType }
+       from '../../../services/Widget/Widget'
+import OSMElementData from '../../OSMElementData/OSMElementData'
+import QuickWidget from '../../QuickWidget/QuickWidget'
+import messages from './Messages'
+
+const descriptor = {
+  widgetKey: 'OSMDataWidget',
+  label: messages.label,
+  targets: [WidgetDataTarget.task],
+  minWidth: 3,
+  defaultWidth: 4,
+  minHeight: 3,
+  defaultHeight: 6,
```

Comment:
I'm not sure what are the best values here

Reply:
What you have currently should be fine since it follows the defaults of similar widgets. It can always be changed if anyone has a problem with the minimum demensions of the widget.